src/Spec2-Morphic-Examples/SpScrollSyncExample.class.st

```diff
@@ -72,8 +72,11 @@ SpScrollSyncExample >> initializePresenters [
 				vResizing: #spaceFill;
 				yourself).
 
-	text whenScrollValueChangedDo: [ :newPosition | self syncMorphPosition ].
-	text text: self loremIpsum
+	text whenScrollValueChangedDo: [ :newPosition | 
+		self traceCr: 'scroll position: ' , newPosition asString. 
+		self syncMorphPosition ].
+	text text: self loremIpsum.
+	text registerEvents.
```

Comment:
registerEvents should not be called, is part of the initialization process.
(and it will introduce leaks if you call it)

Reply:
Ok done. Let me know any further changes.

Reply:
I saw that there is a problem with the [CI](https://github.com/pharo-spec/Spec/actions/runs/3846092231/jobs/6551116716) but I don't quite understand what it is:

```
Could not register the tested repository with Iceberg.

NotFound: object not found - no match for id (8935da6bbbb6fb3767cd26099b6acd5f311ae911)
```